### ISSUE TITLE
Add common busy process and UI for all dialogs

### ISSUE THREAD

**Issue by intervalia** [EvolvedWeb/evowc#74]
**Describe the bug**
The base dialog element <wc-dialog> should have a single way to show a busy spinner, set the aria-busy attribute, and make the entire dialog inert when it is busy.

**Comment by intervalia** [comment]
This is in the next release
